src/components/project/project.resolver.ts

```diff
@@ -312,6 +312,40 @@ export class ProjectResolver {
     );
   }
 
+  @ResolveField(() => SecuredPeriodicReport, { nullable: true })
```

Comment:
```suggestion
  @ResolveField(() => SecuredPeriodicReport)
```
Not nullable, we'll always get an object back for canRead.

Can you add a description here too?